### PR TITLE
Blogpost for LWMIQ 2017-01-30

### PR DESCRIPTION
**Pull request by skateman**

### CODE REVIEW

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Last Week in ManageIQ: Keeping up with the conferences"
+author: Dávid Halász
+date: 2017-01-30 09:39:10 +01:00
```
Comment (moolitayer) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: Didn't see this post in the front page, it was added however after removing the date portion here:
```
date: 2017-01-30
```

Comment (skateman) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: fixed, thanks!

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+published: true
+tags: LWIMIQ
+---
+
+Hi, I am David from the ManageIQ UI team. I will be your guide in this quick recap of what happened in the last week on our premises.
+
+## Featured
+During the weekend, we were promoting ManageIQ on [DevConf.cz](http://devfonf.cz), an amazing conference held in Brno, Czech Republic. Mooli Tayer from the container management gave a really interesting talk about [automating a datacenter using ManageIQ and Ansible](https://www.youtube.com/watch?v=oK8l_u_kHC8). To learn more, check out our [twitter feed](https://twitter.com/manageiq) and look for the [#DevConfCZ](https://twitter.com/search?l=&q=%23devconfcz%20from%3Amanageiq&src=typd) hashtag.
+
+Even if some of us were busy with organizing the conference, we were able to merge 124 PRs into 6 repositories. We are still getting used to the new splitted repositories, so there are a plenty of PRs assuring that everything works well among these repos.
```
Comment (moolitayer) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: s/so there are a plenty of PRS/so there are plenty of PRS

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Ansible is great and we would like to have better integration with our automation. [James](https://github.com/ManageIQ/manageiq/pull/13630) is one of the people working on this very hard. By [migrating](https://github.com/ManageIQ/manageiq/pull/13630) the `ConfigurationManager` of Ansible Tower to the new `AutomationManager` we are one step closer in achieving this goal.
+
+## Fixed
+
+As we are trying to move the VMware provider to its new repo, some paths in requiring helpers weren't been adjusted properly. Thanks to [Adam](https://github.com/agrare) this is now [fixed and working](https://github.com/ManageIQ/manageiq-providers-vmware/pull/2). These things are expected consequences of the moving, but they can be found and fixed quickly.
+
+We are proudly using [Sauce Labs](https://saucelabs.com/) for continuous integration of our Service UI. This feature can be enabled for fork of repository by turning on Travis tests. However, when you forgot your Sauce Labs credentials, the tests were running and failing. Thanks to [Chris Hale](https://github.com/chalettu) this is now [fixed](https://github.com/ManageIQ/manageiq-ui-service/pull/460).
```
Comment (moolitayer) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: s/for fork of repository/for a fork of the repository/

**Comment by JPrause** [comment]
@d-m-u @imtayadeway @syncrou @jrafanie here's the latest blog entry. Can you each review before we merge. Also, I just sent an e-mail regarding editing suggestions. Please have a look at the e-mail before reviewing.

**Comment by d-m-u** [comment]
👍 Looks just spiffing to me.

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+date: 2017-01-30
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi, I am David from the ManageIQ UI team. I will be your guide in this quick recap of what happened in the last week on our premises.
+
+## Featured
+During the weekend, we were promoting ManageIQ on [DevConf.cz](http://devfonf.cz), an amazing conference held in Brno, Czech Republic. Mooli Tayer from the container management gave a really interesting talk about [automating a datacenter using ManageIQ and Ansible](https://www.youtube.com/watch?v=oK8l_u_kHC8). To learn more, check out our [twitter feed](https://twitter.com/manageiq) and look for the [#DevConfCZ](https://twitter.com/search?l=&q=%23devconfcz%20from%3Amanageiq&src=typd) hashtag.
```
Comment (jrafanie) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: > Mooli Tayer from the container management gave

Missing team?

Mooli Tayer from the container management team gave

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+Ansible is great and we would like to have better integration with our automation. [James](https://github.com/ManageIQ/manageiq/pull/13630) is one of the people working on this very hard. By [migrating](https://github.com/ManageIQ/manageiq/pull/13630) the `ConfigurationManager` of Ansible Tower to the new `AutomationManager` we are one step closer in achieving this goal.
+
+## Fixed
+
+As we are trying to move the VMware provider to its new repo, some paths in requiring helpers weren't been adjusted properly. Thanks to [Adam](https://github.com/agrare) this is now [fixed and working](https://github.com/ManageIQ/manageiq-providers-vmware/pull/2). These things are expected consequences of the moving, but they can be found and fixed quickly.
```
Comment (jrafanie) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: > some paths in requiring helpers weren't been adjusted properly

some paths in requiring helpers weren't being adjusted properly.

or just

some paths in requiring helpers weren't adjusted properly.

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+We are proudly using [Sauce Labs](https://saucelabs.com/) for continuous integration of our Service UI. This feature can be enabled for a fork of the repository by turning on Travis tests. However, when you forgot your Sauce Labs credentials, the tests were running and failing. Thanks to [Chris Hale](https://github.com/chalettu) this is now [fixed](https://github.com/ManageIQ/manageiq-ui-service/pull/460).
```
Comment (jrafanie) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: > However, when you forgot your Sauce Labs credentials, the tests were running and failing.

Maybe this:
However, if you forgot your Sauce Labs credentials, the tests would run and fail.

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+## New
+
+Collecting metrics is useful, but storing them in an SQL database is not always *The Right Way*&trade;. [Jason](https://github.com/Fryguy) intrudoced a new model called [ActiveMetrics](https://github.com/ManageIQ/manageiq/pull/13436) that is similar to `ActiveRecord` but it has adapters to databases tuned for collecting metrics in a high frequency.
```
Comment (jrafanie) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: > intrudoced

introduced

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+## Deleted
+
+[Brandon](https://github.com/bdunne) found out that some constants were not used anywhere. These were connected with Automation at some point, but no longer used, so it was safe to [get rid of them](https://github.com/ManageIQ/manageiq/pull/13665).
+
+[Milan](https://github.com/mzazrivec) is making our internationalization support better and better, which means that we can delete more legacy code used to maintain compatibility. A good example of this is the use of `PostponedTranslation` which is no longer required in `UiConstants` so [it has been deleted](https://github.com/ManageIQ/manageiq-ui-classic/pull/241).
+
+## Wrapping up
+This was a quick overview of what have we done last week, but feel free to check out the complete list for each repository. If you want to meet us in person, we will be attending on [FOSDEM](https://fosdem.org/2017/) this weekend in Brussels, Belgium. Search for the people in ManageIQ T-shirts!
```
Comment (jrafanie) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: >we will be attending on [FOSDEM](https://fosdem.org/2017/) this weekend in Brussels, Belgium

I think the 'on' is not needed here.

we will be attending [FOSDEM](https://fosdem.org/2017/) this weekend in Brussels, Belgium

**Comment by skateman** [comment]
Thanks @jrafanie, all fixed!

**Comment by jrafanie** [comment]
LGTM, nice work @skateman !!! 👍

**Comment by JPrause** [comment]
@skateman I think all reviews and edits have been (as well as tests) completed. Is this the case,..if yes, we'll get this merged right away.

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+published: true
+tags: LWIMIQ
+---
+
+Hi, I am David from the ManageIQ UI team. I will be your guide in this quick recap of what happened in the last week on our premises.
+
+## Featured
+During the weekend, we were promoting ManageIQ on [DevConf.cz](http://devfonf.cz), an amazing conference held in Brno, Czech Republic. Mooli Tayer from the container management team gave a really interesting talk about [automating a datacenter using ManageIQ and Ansible](https://www.youtube.com/watch?v=oK8l_u_kHC8). To learn more, check out our [twitter feed](https://twitter.com/manageiq) and look for the [#DevConfCZ](https://twitter.com/search?l=&q=%23devconfcz%20from%3Amanageiq&src=typd) hashtag.
+
+Even if some of us were busy with organizing the conference, we were able to merge 124 PRs into 6 repositories. We are still getting used to the new splitted repositories, so there are plenty of PRs assuring that everything works well among these repos.
```
Comment (imtayadeway) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: Is this supposed to be "split" or is "splitted" a humorous variant?

Comment (imtayadeway) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: Not sure, but is "ensuring" meant here?

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+## Improved
+
+[Eric Winchell](https://github.com/epwinchell) is responsible for having more and more vector graphics in the UI. His plan is to use fonticons where we don't need more than one color and SVG files where colors are necessary. His [last merged PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/249) gets rid of references to PNG files in trees related to Automation and Policies. Hopefully when we are fully converted across the whole UI, we can get rid a hundreds of unused PNG files.
```
Comment (imtayadeway) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: Possibly "our having more and more [...]" or "creating more and more"?

Comment (imtayadeway) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: s/rid a/rid of/

**Comment by imtayadeway** [comment]
@skateman LGTM with some nits. Thanks! :heart:

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
+---
+title: "Last Week in ManageIQ: Keeping up with the conferences"
+author: Dávid Halász
```
Comment (cybette) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: Might be better to use skateman here, as you did for your previous 2 posts. In addition I think there are plans to link author ids to github accounts.

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+date: 2017-01-30
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi, I am David from the ManageIQ UI team. I will be your guide in this quick recap of what happened in the last week on our premises.
```
Comment (cybette) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: Would you like to link to your twitter or github account with "David"?

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+
+## Featured
+During the weekend, we were promoting ManageIQ on [DevConf.cz](http://devfonf.cz), an amazing conference held in Brno, Czech Republic. Mooli Tayer from the container management team gave a really interesting talk about [automating a datacenter using ManageIQ and Ansible](https://www.youtube.com/watch?v=oK8l_u_kHC8). To learn more, check out our [twitter feed](https://twitter.com/manageiq) and look for the [#DevConfCZ](https://twitter.com/search?l=&q=%23devconfcz%20from%3Amanageiq&src=typd) hashtag.
```
Comment (cybette) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: Typo: http://devconf.cz/

For the hashtag link, consider removing "from:" so it includes tweets tweeted by others (e.g. you :smile: )

File: site/_posts/2017-01-30-keeping-up-with-the-conferences.md [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+
+Even if some of us were busy with organizing the conference, we were able to merge 124 PRs into 6 repositories. We are still getting used to the new split repositories, so there are plenty of PRs ensuring that everything works well among these repos.
+
+## Improved
+
+[Eric Winchell](https://github.com/epwinchell) is responsible for creating more and more vector graphics in the UI. His plan is to use fonticons where we don't need more than one color and SVG files where colors are necessary. His [last merged PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/249) gets rid of references to PNG files in trees related to Automation and Policies. Hopefully when we are fully converted across the whole UI, we can get rid of hundreds of unused PNG files.
+
+Ansible is great and we would like to have better integration with our automation. [James](https://github.com/ManageIQ/manageiq/pull/13630) is one of the people working on this very hard. By [migrating](https://github.com/ManageIQ/manageiq/pull/13630) the `ConfigurationManager` of Ansible Tower to the new `AutomationManager` we are one step closer in achieving this goal.
```
Comment (cybette) [inline site/_posts/2017-01-30-keeping-up-with-the-conferences.md]: There are 2 links to the PR. Perhaps the first one is meant to link to [James' github page](https://github.com/jameswnl)?

**Comment by cybette** [comment]
Thanks @skateman for the great post after working all weekend at DevConf.CZ! :+1: :+1: :+1:

**Comment by skateman** [comment]
@cybette you're welcome, and now don't forget to wear your ManageIQ T-shirt at FOSDEM :wink:

**Pull request merged**